package-templates/basic-01/{{project-name | upper_camel_case}}/Cargo.toml.liquid

```diff
@@ -8,6 +8,17 @@ rust-version = "1.64"
 edition = "2021"
 publish = false
 
+[workspace.dependencies]
+# Pin cxx-build to avoid 1.0.187 which requires Rust 1.87
+cxx-build = ">=1, <1.0.187"
+# Standard dependencies used across services
+psibase = { path = "../../../rust/psibase/", version = "{{version}}" }
+psibase_macros = { path = "../../../rust/psibase_macros/", version = "{{version}}" }
+async-graphql = "=7.0.17"
+serde = { version = "1.0.208", features = ["derive"] }
+serde_json = "1.0.125"
+wit-bindgen-rt = { version = "0.30.0", features = ["bitflags"] }
```

Comment:
Why are these changes different to the changes added to all the other user packages?

https://github.com/gofractally/psibase/pull/1494/files

Reply:
ah! great catch. Updated.

Reply:
No, not the `serde_json` version, this entire `[workspace.dependencies]` section is just absent in all the other user packages
e.g. [Branding](https://github.com/gofractally/psibase/blob/c2219a22ff49c722f809b9fd1cf26192b23a0a23/packages/user/Branding/Cargo.toml), [Chainmail](https://github.com/gofractally/psibase/blob/c2219a22ff49c722f809b9fd1cf26192b23a0a23/packages/user/Chainmail/Cargo.toml)

Reply:
oh, derp. thanks.